### ISSUE TITLE
There is no code.co_endlinetable and code.co_columntable in python3.11

### ISSUE THREAD

**Issue by ondrejj** [jackrosenthal/kajiki#79]
There is no code.co_endlinetable and code.co_columntable in python3.11. Upgrading python to 3.11 causes to trash tgext debugbar with exception:
```
File "/home/ondrejj/.local/lib/python3.11/site-packages/kajiki/template.py", line 419, in patch_code_file_lines
    code.co_endlinetable if version_info >= (3, 11) else "REMOVE",
AttributeError: 'code' object has no attribute 'co_endlinetable'
```

Looks that these attributes are missing in python 3.11.

```
Python 3.11.0 (main, Oct 24 2022, 00:00:00) [GCC 12.2.1 20220819 (Red Hat 12.2.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import code
>>> o = code.compile_command("a=1")
>>> o.co_
o.co_argcount         o.co_flags            o.co_names
o.co_cellvars         o.co_freevars         o.co_nlocals
o.co_code             o.co_kwonlyargcount   o.co_positions(
o.co_consts           o.co_lines(           o.co_posonlyargcount
o.co_exceptiontable   o.co_linetable        o.co_qualname
o.co_filename         o.co_lnotab           o.co_stacksize
o.co_firstlineno      o.co_name             o.co_varnames
>>> o.co_
```

**Comment by amol-** [comment]
This is in practice a duplicate of #75 and was fixed in #78

**Comment by jackrosenthal** [comment]
Merged #78, thanks all!
